packages/@okta/vuepress-site/docs/guides/sign-in-with-google/configure-idp/index.md

```diff
@@ -6,13 +6,13 @@ title: Configure Google as an Identity Provider in Okta
 
 2.2. On the main page, click on the **Admin** button in the upper right.
 
-2.3. Hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**.
+2.3. If you use the Developer Console, hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**. If you use the Classic UI, hover over **Security** then click on **Identity Providers**.
```

Comment:
If you use the Developer Console, hover your cursor over **Users** until the menu opens, and then click **Social & Identity Providers**. If you use the Classic UI, hover over **Security**, and then click **Identity Providers**.